ssh.go

```diff
@@ -43,16 +44,23 @@ func (e ErrConnect) Error() string {
 
 // parseHost parses and normalizes <user>@<host:port> from a given string.
 func (c *SSHClient) parseHost(host string) error {
-	c.host = host
+	if !strings.Contains(host, "://") {
+		host = "ssh://" + host
+	}
+
+	info, err := url.Parse(host)
```

Comment:
can we call this `u` or `url`?

Reply:
the object is about host info, can we change the variable name to "h" or "hostInfo"?

Reply:
it's the same name,  "url" and package "net/url".  I'll use the "hostURL" :-)